modules/kms-keyring/test/kms_keyring.ondecrypt.test.ts

```diff
@@ -46,7 +46,7 @@ describe('KmsKeyring: _onDecrypt',
       const clientProvider: any = () => {
         return { decrypt }
         function decrypt ({ CiphertextBlob, EncryptionContext, GrantTokens }: any) {
-          expect(EncryptionContext === context).to.equal(true)
+          expect(EncryptionContext).to.deep.equal(context)
```

Comment:
Nitpick: Since you just ran around doing s/context/encryptionContext/ everywhere else, you might consider doing that here and in the other tests where it's still `context`. I don't think I normally would have even noticed but after reviewing that change in 26 files it stuck out. :P

Reply:
This is valid.  The examples are another case of this.... Sigh.  I agree, but I think I'll come back to this later.  Since this is large enough it will be easier to change words after a stable position.

#160

Reply:
+1, definitely not worth holding this up.